src/test/javascript/spec/component/text-submission-assessment/text-submission-assessment.component.spec.ts

```diff
@@ -121,30 +132,30 @@ describe('TextSubmissionAssessmentComponent', () => {
             studentParticipation: participation,
         }),
     } as unknown as ActivatedRoute;
-    beforeEach(async () => {
+    beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [
-                ArtemisTestModule,
-                ArtemisSharedModule,
-                ArtemisAssessmentSharedModule,
-                AssessmentInstructionsModule,
-                TranslateModule.forRoot(),
-                ArtemisConfirmIconModule,
-                TextSharedModule,
-                RouterTestingModule,
-                ArtemisGradingInstructionLinkIconModule,
-            ],
```

Comment:
Improving test code speed. Nice 👍

Reply:
haha, working with the old suit was like a hell, i had to wait over 15min to run only one test case.